scripts/v2-migration/internal/convert/pg.go

```diff
@@ -0,0 +1,91 @@
+package convert
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/cloudquery/plugin-sdk/schema"
+)
+
+func SchemaTypeToPg(t schema.ValueType) (string, error) {
```

Comment:
This was copied from the postgresql package, because I can't import from an `internal` package elsewhere.